guide/payments/transactions.md

```diff
@@ -88,6 +88,19 @@ In a Bitcoin wallet, funds are often not held in a single address, but more comm
 
 Likewise, you need to specify the destination address or addresses for the transaction. These are called *outputs*. should there be more bitcoin in the inputs than are needed for the payment, a new address will be created in your wallet for the remaining change, often called a *change output*.
```

Comment:
```suggestion
Likewise, you need to specify the destination address or addresses for the transaction. These are called *outputs* should there be more bitcoin in the inputs than are needed for the payment, a new address will be created in your wallet for the remaining change, often called a *change output*.
```

Just sneaking in this typo fix.

Reply:
I think the period should stay there. I think it should be:

```
These are called *outputs*. Should there be more...
```

Reply:
yes, you are correct sir. i blame the lack of sleep.